packages/host/app/services/matrix-service.ts

```diff
@@ -393,6 +397,13 @@ export default class MatrixService extends Service {
     }
   }
 
+  private addTools = (attachedOpenCard: CardDef, patchSpec: any) => {
```

Comment:
Would `toolsForAdding` be a better name? I was confused and thought this is "adding tools" to something